mlir/test/Dialect/Transform/test-interpreter-external-symbol-def-invalid.mlir

```diff
@@ -0,0 +1,14 @@
+// RUN: mlir-opt %s
```

Comment:
Do we need a run line here?
If `lit` doesn't skip this file, maybe we could move all the "include" files in a sub-dir and tell lit to ignore these files.

Edit: I see that we have patterns of files like that already (e.g., `mlir/test/Dialect/Transform/test-interpreter-external-symbol-def.mlir`). I don't particularly like this kind of "fake" run line, but maybe we are actually testing something here?
@ftynse what do you think we should do?

Reply:
I copied this from other files and speculated that this is done to (1) make lit not complain and (2) maybe make a test fail if the syntax of this specific file gets out of date rather than relying a test of another file?